packages/core/src/index.ts

```diff
@@ -35,8 +31,6 @@ type Enumerate<N extends number, Acc extends number[] = []> = Acc['length'] exte
 
 export type HTTPMethodRange<F extends number, T extends number> = Exclude<Enumerate<T>, Enumerate<F>>;
 
-export { getJSONSchemaDefaults, parseResponse, prepareAuth, prepareParams, prepareServer };
-
 export default class APICore {
```

Comment:
This default export alongside the named TS type exports is still causing issues with `attw`:

```
> @readme/api-core@7.0.0-alpha.3 attw
> attw --pack --format table-flipped


@readme/api-core v7.0.0-alpha.3

Build tools:
- typescript@^5.2.2

❓ The JavaScript appears to set both module.exports and module.exports.default for improved compatibility, but the types only reflect the latter (by using export default). This will cause TypeScript under the node16 module mode to think an extra .default property access is required, which will work at runtime but is not necessary. These types export = an object with a default property instead of using export default. https://github.com/arethetypeswrong/arethetypeswrong.github.io/blob/main/docs/problems/MissingExportEquals.md

💀 Import failed to resolve to type declarations or JavaScript files. https://github.com/arethetypeswrong/arethetypeswrong.github.io/blob/main/docs/problems/NoResolution.md


┌──────────────────────────────────────┬───────────────────────┬───────────────────────┬───────────────────┬───────────┐
│                                      │ node10                │ node16 (from CJS)     │ node16 (from ESM) │ bundler   │
├──────────────────────────────────────┼───────────────────────┼───────────────────────┼───────────────────┼───────────┤
│ "@readme/api-core"                   │ ❓ Missing `export =` │ ❓ Missing `export =` │ 🟢 (ESM)          │ 🟢        │
├──────────────────────────────────────┼───────────────────────┼───────────────────────┼───────────────────┼───────────┤
│ "@readme/api-core/errors/fetchError" │ 💀 Resolution failed  │ 🟢 (CJS)              │ 🟢 (ESM)          │ 🟢        │
├──────────────────────────────────────┼───────────────────────┼───────────────────────┼───────────────────┼───────────┤
│ "@readme/api-core/package.json"      │ 🟢 (JSON)             │ 🟢 (JSON)             │ 🟢 (JSON)         │ 🟢 (JSON) │
└──────────────────────────────────────┴───────────────────────┴───────────────────────┴───────────────────┴───────────┘
```

Everything _seems_ to work with the dist but it still worries me. I attempted to refactor the TS type exports out into a `types.ts` file, which satisfies `attw` but using that types file is another story:

```diff
diff --git a/packages/core/package.json b/packages/core/package.json
index 5d16e2b..4e95761 100644
--- a/packages/core/package.json
+++ b/packages/core/package.json
@@ -13,6 +13,10 @@
       "require": "./dist/errors/fetchError.cjs",
       "import": "./dist/errors/fetchError.js"
     },
+    "./types": {
+      "require": "./dist/types.d.cts",
+      "import": "./dist/types.d.ts"
+    },
     "./package.json": "./package.json"
   },
   "main": "dist/index.cjs",
```

![Screen Shot 2023-10-09 at 5 37 16 PM](https://github.com/readmeio/api/assets/33762/000f8112-a726-4d59-9eca-d58dfa793c56)